metadata/static/jenkins/overview.md

```diff
@@ -6,7 +6,7 @@ ways to define your build pipelines and integrating with a large number of testi
 
 ## Support Level
 - Mixed workload tested
-  - 100 Jenkins Masters
+  - 250 Jenkins Masters
```

Comment:
We tested upto 500 jenkins masters but after 250 masters, the launch duration exceeded 600 seconds (which was used as a cutoff for DC/OS jenkins mwt)

Reply:
Thanks for testing this :+1:

Reply:
:heavy_check_mark: